README.md

```diff
@@ -19,6 +19,28 @@ The library offers:
 ## Intents
 Songbird's gateway functionality requires you to specify the `GUILD_VOICE_STATES` intent.
 
+## Dependencies
+
+Songbird needs a few system dependencies before you can use it.
+
+- Opus - Audio codec that Discord uses.
+You can install the library with `apt install libopus-dev` on Ubuntu or `pacman -S opus` on Arch Linux.
+If you do not have it installed it will be built for you. However, you will need a C compiler and the GNU autotools installed.
+Again, these can be installed with `apt insall build-essential autoconf automake libtool m4` on Ubuntu or `pacman -S base-devel` on Arch Linux.
```

Comment:
/s/insall/install

Also, we should mention that Windows (MSVC) should just work, as audiopus will link against a prebuilt dll.